### DISCUSSION TITLE
removing handcrafted terminal solutions; better pseudoterminal/afterlife handling

### DISCUSSION THREAD

**Discussion by sbenthall** [econ-ark/HARK#1662] [General]
Currently, both handcrafted terminal and pseudo-terminal models are supported.

The handcrafted terminal functionality should be deprecated and all models should have better, more robust "pseudo-terminal", "afterlife" functionality for the last period.

**Comment by mnwhite** [comment]
Can you elaborate on this? Are you proposing that all models should use what I've called "pseudo-terminal" periods? CDC hates that these exist at all.

**Comment by sbenthall** [comment]
I don't recall the context for this issue and would be fine if it were closed as wontfix.

I suppose there's a question of how HARK 1.0 would deal with this.
